src/helics/core/BrokerBase.cpp

```diff
@@ -475,9 +489,11 @@ void BrokerBase::queueProcessingLoop ()
             }
             messagesSinceLastTick = 0;
             // reschedule the timer
+#ifndef HELICS_DISABLE_ASIO
```

Comment:
```suggestion
			#ifndef HELICS_DISABLE_ASIO
```

Matching the indentation for the #endif

Reply:
There is a clang-format option for formatting indentation of pragmas in the newer version of clang-format.  I am hoping when we get that automated the pragmas such as this will be consistent